starwars/src/components/SWCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+
+
+const Card = styled.div`
+  width: 350px;
+  height: 250px;
+  font-size: 15px;
+  color: black;
+  background: sandybrown;
+  margin: 10px;
+  border-radius: 30px;
+  
+  &:hover {
+    color: steelblue;
+    background: wheat;
+
+ }
+`;
+
+const SWCard = props => {
+    return (
+    <div className="star-card" key={props.id}>
+        <Card>
+          <h2>{props.name}</h2>
```

Comment:
Note that you can make your code more DRY by passing the whole object instead of each individual key into the component and destructuring, like this: `const { height, gender, weight, birth_year } = char` . this allows you to skip prepending props onto each variable